docs/maintainers/releasing.md

```diff
@@ -1,77 +1,72 @@
-so `v6.0.0-alpha.7` goes to `v6.0.0-alpha.8`.
-
-When a particular commit is chosen as a release,
-we create an [annotated tag][tag] that names the release,
-and list the major changes since the previous release.
-An annotated tag has a message (like a commit),
-and is *not* the same as a lightweight tag.
-Create one with `git tag -a v6.0.0-alpha.7`.
-Our convention is to prepend the `v` to the semantic version.
-The summary (first line) of the annotated tag message should be the full release title,
-e.g. 'v6.0.0-alpha.7 release of PowerShell'.
-
-While creating a release, it is advised to make a new branch such that
-necessary documentation updates and hot fixes can be made,
-without having to include all changes made to master.
-This release branch can be reviewed by the normal PR process.
-
-When the annotated tag is finalized, push it with `git push --tags`.
-GitHub will see the tag and present it as an option when creating a new [release][].
-Start the release, use the annotated tag's summary as the title,
-and save the release as a draft while you upload the binary packages.
-
-Just as important as creating the release is updating the links on our readme,
-and the package names in the installation instructions.
-The AppVeyor build number should also be incremented.
-
-When creating the packages, please ensure that the file path does not contain user names.
+# PowerShell Core Releasing Process
+
+## Release Steps
+
+1. Create a branch named `release` in `PowerShell/PowerShell` repository. All release related changes should happen in this branch.
+1. Prepare packages
+   - [Build release packages](#building-packages)
+   - Sign the MSI packages and DEB/RPM packages.
+   - Install and verify the packages. (_could be automated for at least Linux packages_)
+1. Update documentation, scripts and Dockerfiles
+   - Summarize the change log for the release. It should be reviewed by PM(s) to make it more user-friendly.
+   - Update [CHANGELOG.md](../../CHANGELOG.md) with the finalized change log draft.
+   - Update other documents and scripts to use the new package names and links.
+1. Verify the release Dockerfiles. (_could be automated_)
+1. Publish Linux packages to Microsoft YUM/APT repositories.
+1. [Create NuGet packages](#nuget-packages) and publish them to [powershell-core feed][ps-core-feed].
+1. [Create the release tag](#release-tag) and push the tag to `PowerShell/PowerShell` repository.
+1. Merge the `release` branch to `master` and delete the `release` branch.
+1. Publish the release in Github.
+1. Trigger the release docker builds for Linux and Windows container images.
+   - Linux: push a branch named `docker` to `powershell/powershell` repository to trigger the build.
+     Delete the `docker` branch once the builds successful at [powershell docker hub](https://hub.docker.com/r/microsoft/powershell/builds/).
+   - Windows: queue a new build from [PowerShellWindowsDocker](https://dscsolutions.visualstudio.com/PowerShellWindowsDocker).
```

Comment:
We can discuss how to connect the docs at two repositories. For now, I will remove the internal URL.